test/TestAsset/ConfigPostProcessor/NonAliasedServiceConfiguration.php

```diff
@@ -13,10 +13,13 @@
             'Zend\Expressive\OtherInterface' => 'OtherService',
             'OtherService' => 'Mezzio\OtherInterface',
             'MyAlias' => 'Zend\Expressive\AliasChain',
+            'Mezzio\TwoInterface' => 'middle-alias',
+            'middle-alias' => 'Zend\Expressive\TwoInterface',
         ],
         'invokables' => [
             'Zend\Expressive\AliasInterface' => 'MyNamespace\InvokableClass',
             'Zend\Expressive\AliasChain' => 'MyNamespace\OtherInvokableClass',
+            'Zend\Expressive\TwoInterface' => 'MyNamespace\TwoInvokableClass',
```

Comment:
This seems.... really convoluted. As written, it looks like something that would never happen ("why would somebody mix both Mezzio and Expressive code?").

But the point here is when mixing configuration from a migrated project with a 3rd party library. What I think we need to do is make this more clear, maybe with something like:

```php
'aliases' => [
    Mezzio\Router\RouterInterface::class => 'router',
    'router' => Zend\Expressive\Router\RouterInterface::class,
],
'invokables' => [
    Zend\Expressive\Router\RouterInterface::class => MyNamespace\InvokableRouterClass::class,
],
```

This makes it a little more clear that the `aliases` include both an alias to a generic service name, as well as that generic service name to an Expressive interface, with the interface service definition being handled in invokables.